test/dotnet-migrate.Tests/GivenThatIWantToMigrateSolutions.cs

```diff
@@ -204,11 +256,10 @@ private void MigrateAndBuild(string groupName, string projectName, [CallerMember
                 .Execute($"restore \"{solutionRelPath}\"")
                 .Should().Pass();
 
-            //ISSUE: https://github.com/dotnet/cli/issues/5205
```

Comment:
If it does actually work (you re-enable it), then there is another test to re-enable that is mentioned in one of the issues you reference.